### ISSUE TITLE
Kicked out of instance - couldn't teleport back in

### ISSUE THREAD

**Issue by Gaugeoak** [gamefreedomgit/Maelstrom#771]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
Started Blackrock dungeon and it kicked us out back to where we were before entering dungeon.  Couldn't teleport back in and received an error "you do not have a valid teleport location" see pic attached
**How to reproduce:**

<img width="1581" alt="Blackrock Tele Issue" src="https://user-images.githubusercontent.com/82425397/132999027-e69ed4c8-05a0-4a38-942f-da62d6982b49.png">

**How it should work:**
Should be able to do the dungeon without being kicked out.  Should be able to tele back in using the green eye

**Database links:**

**Comment by UltraNix** [comment]
Cannot reproduce it.

**Comment by Andkoo** [comment]
Got the same issue right now. The entire team got the message with no valid tp location and there was no green eye on the minimap to tp into the dungeon. Tried a relog, didn't help. How I got there:
1. lvl 84 character, did Throne of the Tides
2. our group decided to requeue after we completed it
3. the system picked ToT again
4. as soon as we clicked Enter Dungeon we got the "no valid tp location" message
5. we tried to tp out and in
6. the instance was empty (completed)
7. we kicked each others to prevent deserter
8. I requeued

I'm not 100% sure but the system probably picked ToT again and that's where it all went wrong.
